### ISSUE TITLE
TransformLogLikelihood terminology

### ISSUE THREAD

**Issue by simonbyrne** [tpapp/ContinuousTransformations.jl#8]
Thanks for the package! It has a lot of really neat features.

My one nitpick is that `TransformLogLikelihood` is not really that accurate of a name. As I understand it, it accounts for the Jacobian term when doing the change-of-variables. But the Jacobian term is due to the fact that it's a density; likelihoods by themselves are parametrisation invariant (e.g. maximum likelihood estimates won't change).

I would suggest `TransformLogDensity` (since you don't need a likelihood at all, e.g. you could just be sampling from an arbitrary distribution).

**Comment by tpapp** [comment]
Good point. I am working on a new package that will replace this one, so I opened an issue there.
https://github.com/tpapp/TransformVariables.jl/issues/1
